src/main/java/dk/kb/present/api/v1/impl/DsPresentApiServiceImpl.java

```diff
@@ -21,6 +32,30 @@
 public class DsPresentApiServiceImpl extends ImplBase implements DsPresentApi {
     private static final Logger log = LoggerFactory.getLogger(DsPresentApiServiceImpl.class);
 
+    // License handling in DsPresentApiServiceImpl as tokens from the request will be needed later on
+    private static final String LICENSE_URL_KEY = "config.licensemodule.url";
+    private static final String LICENSE_ALLOWALL_KEY = "config.licensemodule.allowall";
```

Comment:
Just a comment: It would be so pretty if we removed the redundant `config.` keys from config files.